addons/io_scene_gltf2/blender/exp/gltf2_blender_gather_animations.py

```diff
@@ -198,6 +198,16 @@ def __get_blender_actions(blender_object: bpy.types.Object,
         if blender_object.animation_data.action is not None:
             blender_actions.append(blender_object.animation_data.action)
             blender_tracks[blender_object.animation_data.action.name] = None
+        
+        #Export all actions outside of NLA track
+        if export_settings['gltf_all_actions'] is True:
+            for action in bpy.data.actions:
+                if action is None:
```

Comment:
Is this possible?

Reply:
Ah, the `None` check is just overly cautious, probably doesn't need to be there, don't think it would cause issues whether it's kept or edited out, would it?

Reply:
No, but no reason to keep it if it never runs.